### ISSUE TITLE
Windows only CSV import issue for non-Latin languages

### ISSUE THREAD

**Issue by pcenov** [learningequality/kolibri#9060]
## Observed behavior
When I change the UI language to a non-Latin one such as Greek or Bulgarian, export a CSV file and immediately attempt to import the exported file without modifying it in any way I see a Loading... screen and the import never finishes. There's no indication what is the problem and no error in the console. Restarting Kolibri doesn't solve the issue and it keeps saying Loading....
In other cases where I am attempting to import users I get a 'charmap codec can't decode byte 0x81 in position 57: character maps to <undefined>' error.
I tested this in Windows 10 on a physical device and Win7 and Win10 on virtual machines so you should be able to replicate this issue consistently.

## Expected behavior
Fully functional CSV import functionality for all supported languages in Windows.

## Steps to reproduce
1. Install [Kolibri 0.15](https://learningequality.org/r/kolibri-windows-setup-latest)
2. Setup a facility and go to Facility>Data
3. Change the language to Greek
4. Click on "Generate new user CSV file" and then 'Download CSV"
5. Click "Import" select the exported CSV file without even opening or modifying it and attempt to import it. 

## Screenshots
![2022-01-28_17-04-41](https://user-images.githubusercontent.com/79847249/151571811-cb9794a0-13d5-45b8-b170-69961a6582cc.png)
![2022-01-27_17-21-43](https://user-images.githubusercontent.com/79847249/151571903-07f1218c-642d-433e-8000-a6d00259b067.png)

## Logs
[kolibri.zip](https://github.com/learningequality/kolibri/files/7959701/kolibri.zip)
[Home Facility for a_users_20220128_165723.csv](https://github.com/learningequality/kolibri/files/7959708/Home.Facility.for.a_users_20220128_165723.csv)

## Context

 * Kolibri version: 0.15
 * Operating system: Windows 10, Windows 7
 * Browser: Chrome

**Comment by pcenov** [comment]
@radinamatic

**Comment by jredrejo** [comment]
@pcenov could you:

- run `kolibri manage bulkimportusers  file.csv` with one of the csv that are failing and paste the output here?
- set your windows in one of the non-latin languages and check if it fails too? (I'm not sure if that's possible, if not , forget about it)
Thanks

**Comment by radinamatic** [comment]
Our Windows testing VMs come preloaded with ~20 language packs of those used in Kolibri (some of our languages are not even available on Windows 😉 ), so changing the UI display language to test something is doable.

**Comment by pcenov** [comment]
Hi @jredrejo here's the output when I run the `kolibri manage bulkimportusers file.csv` command: 
```
C:\Users\IEUser>kolibri manage bulkimportusers C:\Users\IEUser\.kolibri\file.csv
[37mINFO     Option DEBUG in section [Server] being overridden by environment variable KOLIBRI_DEBUG[0m
[37mINFO     Option DEBUG_LOG_DATABASE in section [Server] being overridden by environment variable KOLIBRI_DEBUG_LOG_DATABASE[0m
[37mINFO     Option RUN_MODE in section [Deployment] being overridden by environment variable KOLIBRI_RUN_MODE[0m
[37mINFO     Running Kolibri with the following settings: kolibri.deployment.default.settings.base[0m
[37mINFO     Importing 'tasks' module from django apps[0m
[37mINFO     Invoking command bulkimportusers C:\Users\IEUser\.kolibri\file.csv[0m
CommandError: File errors: ["Error trying to read csv file: 'charmap' codec can't decode byte 0x81 in position 54: character maps to <undefined>"]
```
As suggested by you I tested this on a Windows VM with Bulgarian interface , exported the csv in Bulgarian, didn't modify anything in the CSV file and when I attempted to import it it still fails.

**Comment by jredrejo** [comment]
Thanks you both (and thanks to the vm machine Radina sent me some weeks ago) I think I have fixed the problem.
I've added the fix to #9039 to try to have it in 0.15.x asap. 
@pcenov please, check it with the latest changes in #9039 if you have a chance
